tests/auxiliary/minicore.rs

```diff
@@ -13,7 +13,8 @@
 //! <https://github.com/rust-lang/rust/blob/c0b5cc9003f6464c11ae1c0662c6a7e06f6f5cab/compiler/rustc_codegen_cranelift/example/mini_core.rs>.
 // ignore-tidy-linelength
 
-#![feature(no_core, lang_items, rustc_attrs)]
+#![feature(no_core, lang_items, rustc_attrs, decl_macro)]
+#![feature(asm_experimental_arch)]
```

Comment:
This means that minicore will no longer be available for sparc, sparc64, and xtensa, right?

Reply:
@jieyouxu

Reply:
Good point, let me double-check.

Reply:
Oh @taiki-e do you mean because sparc, sparc64, and xtensa don't support `asm_experimental_arch`? In that case, I'll revert the `asm_experimental_arch` addition and put the asm stubs back to the tests.

Reply:
Reverted the `asm_experimental_arch` addition to `minicore` and kept it in tests, good catch, thanks!

Reply:
Sorry, I think previously it was not possible to enable it on architectures that did not support inline asm, but that seems to have been fixed now.
https://godbolt.org/z/b73j6Keeh

Reply:
That's fine, moved it back to `minicore` so tests don't have to repeat it.